src/zip.c

```diff
@@ -381,7 +389,7 @@ int zip_entry_close(struct zip_t *zip) {
     tdefl_status done;
     mz_uint16 entrylen;
     time_t t;
-    struct tm *tm;
+    struct tm tmb, *tm = (struct tm*)&tmb;
```

Comment:
we do we need `tmb`, here? it's not used anywhere.

Reply:
this use is defined _MSC_VER, needed around defined